### ISSUE TITLE
Request for Acknowledgment of Contributions 

### ISSUE THREAD

**Issue by calrloco** [manulthanura/Positivus#1]
I saw the  theme on Astro Themes great job bringing it forward! I originally developed the initial concept and most of the components. It’s exciting to see it evolve!
Could you add me to the contributors list? It’d be a fair recognition of the groundwork I laid.

**Comment by manulthanura** [comment]
Thank you for letting me know about your work and the initial concept! It's great to see someone else inspired by the same [Figma design](https://www.figma.com/community/file/1230604708032389430/positivus-landing-page-design). I coded my theme based on that design as well. 

I just saw your [repository](https://github.com/calrloco/positivus), and it looks like you've got a good start on the theme! Especially some components that I plan to improve on in the future!

Since we both share a vision for this theme, I'd love to explore collaborating! Maybe by combining our efforts, we can create an even more impressive version. So send me a message on [Linkedin](https://lk.linkedin.com/in/manulthanura).
